src/portalbackend/PortalBackend.Migrations/Migrations/20231030161356_CPLP-3317-viewTableImplementations.cs

```diff
@@ -0,0 +1,85 @@
+/********************************************************************************
+ * Copyright (c) 2021, 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Org.Eclipse.TractusX.Portal.Backend.PortalBackend.Migrations.Migrations
+{
+    /// <inheritdoc />
+    public partial class CPLP3317viewTableImplementations : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql(@"CREATE VIEW portal.company_users_view AS
+                SELECT
+                    c.id as company_id,
+                    c.name as company_name,
+                    cu.firstname as firstname,
+                    cu.lastname as lastname,
```

Comment:
Since the properties in CompanyUsersView are call FirstName and LastName they will be made to first_name and last_name from ef, therefore the query needs to be adjusted to work correctly.

```suggestion
                    cu.firstname as first_name,
                    cu.lastname as last_name,
```